core/storage/config/gae_models_test.py

```diff
@@ -52,3 +52,102 @@ def test_commit(self):
         retrieved_model2 = config_models.ConfigPropertyModel.get_version(
             'config_model1', 2)
         self.assertEqual(retrieved_model2.value, 'd')
+
+
+class PlatformParameterModelUnitTests(test_utils.GenericTestBase):
+    """Test PlatformParameterModel class."""
+
+    def test_get_deletion_policy_is_not_applicable(self):
+        self.assertEqual(
+            config_models.PlatformParameterModel.get_deletion_policy(),
+            base_models.DELETION_POLICY.NOT_APPLICABLE)
+
+    def test_get_export_policy_is_not_applicable(self):
```

Comment:
LGTM! 

Just one small comment here. I don't think this test is necessary since we already test to ensure that all models specify the appropriate export policy in core.storage.storage_models_test.

What do you think @seanlip?

Reply:
Hm, I'm fine either way. Thanks for reviewing!

Reply:
I think keeping this test might make it more clear, let's just keep it here. :)